src/cascadia/TerminalApp/LeafPane.idl

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "IPane.idl";
+
+namespace TerminalApp
+{
+    enum Borders2
+    {
+        None = 0,
+        Top = 1,
+        Bottom = 2,
+        Left = 4,
+        Right = 8
+    };
+
+    [default_interface] runtimeclass LeafPane : Windows.UI.Xaml.Controls.UserControl, IPane
+    {
+        LeafPane(Guid profile, Microsoft.Terminal.TerminalControl.TermControl con, Boolean active);
+        Microsoft.Terminal.TerminalControl.TermControl GetTerminalControl();
+        Guid GetProfile();
+        Boolean WasLastFocused();
+        UInt16 Id;
+        Borders2 Borders;
+        void _UpdateBorders();
+        void SetActive();
+        void Close();
+        void UpdateBorderWithClosedNeighbor(LeafPane closedNeighbor, Microsoft.Terminal.Settings.Model.ResizeDirection neighborDirection);
+        LeafPane Split(Microsoft.Terminal.Settings.Model.SplitState splitType, Single splitSize, Guid profile, Microsoft.Terminal.TerminalControl.TermControl con);
```

Comment:
nit: to make this look cleaner, can you...
- replace `Get<thing>()` with `<Thing> { get; }`
- separate out the properties and functions into separate lists (like, list out the getters first, then new line, then the functions)